src/Math/Direction2.cs

```diff
@@ -245,6 +245,12 @@ public override int GetHashCode()
 
         #endregion
 
+        #region tostring
+
+        public override string ToString() => $"{Radians} rad";
```

Comment:
I think I'd rather just have sensible defaults, especially since ToString() is what will be used to show debug/logging information, and custom overloads won't really be picked up by anything unless you call them specifically.